src/tests/ftest/soak/smoke.yaml

```
@@ -173,6 +175,33 @@ fio_smoke:
         mount_dir: "/tmp/daos_dfuse/fio/"
 daos_racer:
     runtime: 120
+vpic_smoke:
+    job_timeout: 10
+    nodesperjob:
+        - 1
+    taskspernode:
+        - 1
+    cmdline: "/home/mjean/vpic-install/bin/harris.Linux"
+    posix: True
+    workdir: "/tmp/daos_dfuse/vpic/"
+    module: "mpi/latest"
+    dfuse:
+        mount_dir: "/tmp/daos_dfuse/vpic/"
+    oclass:
+        - "EC_2P1GX"
+lammps_smoke:
+    job_timeout: 10
+    nodesperjob:
+        - 1
+    taskspernode:
+        - 1
```

Curious, why not this?
```suggestion
    nodesperjob: 1
    taskspernode: 1
```

---

It is a list;   for smoke; the test only uses -np 1 but for stress it is a list of nodesperjobs and taskperjobs to poduce jobs with different -np